### ISSUE TITLE
How to fix the startup error messages related to settings

### ISSUE THREAD

**Issue by E3V3A** [alievk/avatarify#101]
I get the following error messages when starting up:

```
load checkpoints..
Warning: No faces were detected.

C:\Users\xxxx\miniconda3\envs\avatarify\lib\site-packages\torch\nn\functional.py:2705: UserWarning: 
Default grid_sample and affine_grid behavior has changed to align_corners=False since 1.3.0. 
Please specify align_corners=True if the old behavior is desired. 
See the documentation of grid_sample for details.
  warnings.warn("Default grid_sample and affine_grid behavior has changed "

C:\Users\xxxx\miniconda3\envs\avatarify\lib\site-packages\torch\nn\functional.py:2503: UserWarning: 
Default upsampling behavior when mode=bilinear is changed to align_corners=False since 0.4.0.
Please specify align_corners=True if the old behavior is desired. 
See the documentation of nn.Upsample for details.
  warnings.warn("Default upsampling behavior when mode={} is changed "

C:\Users\xxxx\miniconda3\envs\avatarify\lib\site-packages\torch\nn\functional.py:1351: UserWarning: 
nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.
  warnings.warn("nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.")
Loading StyleGAN avatar...
```

So where do I set these items:

* `align_corners=True`?
* `mode=bilinear`?
* ` torch.sigmoid`?

**Comment by alievk** [comment]
These are warnings, not errors

**Comment by E3V3A** [comment]
@alievk  
true, but i still want to get rid of them. It freaks people out, for absolutely no good reason.
I also would like to better understand how they can be fixed.
